test/test_utils.erl

```diff
@@ -120,6 +120,115 @@ match_map(Expected, Got) when is_map(Got) ->
 match_map(_Expected, _Got) ->
     {false, not_map}.
 
+frame_packet_uplink(MType, PubKeyBin, SigFun, DevAddr, FCnt, Options) ->
```

Comment:
@macpie now that we're not a full LNS, I was hoping we could get out of the business of constructing fully formed packets. Mostly we only care about the EUI and Devaddr, then we could fill the rest with junk.

Thoughts?

Reply:
I want to stay as close to real packets as possible so I would like to keep that